README.md

```diff
@@ -20,35 +22,20 @@ faster than other packages according to public benchmarks.
 ## Table of contents
 
 * [Installation](#installation)
-* [Hello World](#hello-world)
-* [API reference](#api-reference)
-* [Walking\-through example in Go](#walking-through-example-in-go)
-* [Help](#help)
-* [Usage](#usage)
-* [Schema](#schema)
-* [Custom (un)packing and typed selects and function calls](#custom-unpacking-and-typed-selects-and-function-calls)
-* [Options](#options)
-* [Working with queue](#working-with-queue)
-* [Tests](#tests)
+* [Documentation](#documentation)
+  * [API reference](#api-reference)
+  * [Walking\-through example](#walking-through-example)
+* [Contributing](#contributing)
 * [Alternative connectors](#alternative-connectors)
 
 ## Installation
 
-We assume that you have Tarantool version 1.6 and a modern Linux or BSD
+We assume that you have Tarantool version 1.6+ and a modern Linux or BSD
 operating system.
 
```

Comment:
You need a current version of `go`, version 1.3 or higher (use
`go version` to check the version number). Do not use `gccgo-go`.

**Note:** If your `go` version is older than 1.3 or if `go` is not installed, 
download and run the latest tarball from [golang.org](https://golang.org/dl/):

Reply:
Updated.